### DISCUSSION TITLE
Missing withCredentials after Upgrading from 6.2.0 to 7.3.

### DISCUSSION THREAD

**Discussion by mleister97** [wojtekmaj/react-pdf#1585] [Q&A] [answered]
### Before you start - checklist

- [X] I followed instructions in documentation written for my React-PDF version
- [X] I have checked if this bug is not already reported
- [X] I have checked if an issue is not listed in [Known issues](https://github.com/wojtekmaj/react-pdf/wiki/Known-issues)
- [ ] If I have a problem with PDF rendering, I checked if my PDF renders properly in [PDF.js demo](https://mozilla.github.io/pdf.js/web/viewer.html)

### Description

After upgrading from version 6.2.0 to 7.3.3 of the library, I noticed that the `withCredentials` flag is suddenly missing. I have thoroughly checked the Changelogs and the Upgrade guide for version 7.3.3, but there is no mention of this change. The withCredentials functionality was essential for my use case, and its absence is causing issues with my application's authentication flow. I would appreciate any guidance or clarification on why this change was not documented and how I can regain the withCredentials capability in the latest version of the library. Thank you!

### Steps to reproduce

Upgrade to newest version

### Expected behavior

The flag should still be there or at least mentioned somewhere. Is it no longer needed?

### Actual behavior

No information about the removed flag

**Comment by wojtekmaj** [comment, marked as answer]
Yep `withCredentials` flag was not really a part of `file`, so it was moved to `options` where it belongs. 

Well technically, it wasn't even moved, just better documented, because `file` and `options` get merged before passing to PDF.js :D

Please mind that `withCredentials` is mentioned in the docs. :)

**Reply by mleister97** [reply]
Thank you so much
